test/gtest/perf_helper.hpp

```diff
@@ -110,7 +110,8 @@ struct PerfHelper
         }
 
         // If the file was just created (i.e., its size is 0), write the header.
-        if(std::filesystem::file_size(filename) == 0)
```

Comment:
No objections but why not use `fs::file_size` since we have `#include <miopen/filesystem.hpp>`

Reply:
Fair point. I'll update.